### PR TITLE
Aux modules spelling

### PR DESCRIPTION
**Pull request by h00die**
Cleanup of spelling across the `modules/auxiliary/` folder. I avoided fixing apostrophes.

Most of these changes are in comments and in user prompts

Also ran rubocop against the `telegram/send_message` module as it was the only one `msf_tidy` was complaining about

## Verification

- [ ] manual review
- [ ] run some of the modules to make sure they work
- [ ] use some things to make sure they dont break

### CODE REVIEW

**Comment by h00die** [comment]
There was a few modules which had spelling errors in regexes and matches of various sorts. I left these thinking it may be trying to match against software which had the spelling issue thus it needs to keep it as is.  Could be that, or some branches of code never worked, who knows :)

File: modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def run
       }
     })
 
-    # Yes, "sucess" is really mispelt, as is "Servelet" ... !
+    # Yes, "success" is really misspelt, as is "Servelet" ... !
```
Comment (adfoster-r7) [inline modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb:65]: Needs reverted

**Comment by adfoster-r7** [comment]
## Release Notes

Fix spelling mistakes in Metasploit's auxiliary modules

File: modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb (hunk 1 above)
Comment (h00die) [reply modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb]: when its all spelled out like that yea, good catch!

**Pull request merged**